query-service-impl/src/main/java/org/hypertrace/core/query/service/ExecutionContext.java

```diff
@@ -171,6 +172,10 @@ private void extractColumns(List<String> columns, Expression expression) {
         break;
       case FUNCTION:
         Function function = expression.getFunction();
+        // ignore argument list for COUNT function as it is just converted to COUNT(*) at the end
+        if (function.getFunctionName().equalsIgnoreCase(QUERY_FUNCTION_COUNT)) {
```

Comment:
In some cases aggregated table is not being picked just because count is on a column not part of aggregated table. Since column name used in count function is never used(also it is irrelevant to the final outcome), therefore I have added logic to just ignore it.